lib/src/main/java/com/auth0/jwt/verification/VerificationAndAssertion.java

```diff
@@ -137,14 +137,14 @@ private static void assertDateIsPast(Date date, long leeway, Date today) {
         }
     }
 
-    private static void assertValidAudienceClaim(List<String> audience, List<String> value) {
-        if (audience == null || !audience.containsAll(value) || audience.size() != value.size()) {
+    private static void assertValidAudienceClaim(List<String> actual, List<String> expected) {
+        if (expected == null || !expected.containsAll(actual)) {
             throw new InvalidClaimException("The Claim 'aud' value doesn't contain the required audience.");
         }
     }
 
-    private static void assertValidIssuerClaim(List<String> issuer, List<String> value) {
-        if (issuer == null || !issuer.containsAll(value) || issuer.size() != value.size()) {
+    private static void assertValidIssuerClaim(List<String> actual, List<String> expected) {
+        if (expected == null || !expected.contains(actual.get(0))) {
```

Comment:
why only the first issuer??  what about the others?  if `iss` claim is going to be taken as a String rather than an Array, then change the method signature instead.

Reply:
so should we have it such that the actual issuer is contained within the expected?  there can be multiple expected issuers, but as long as the actual is contained, we're fine?